topics/contributing/tutorials/github-contribution/tutorial.md

```diff
@@ -52,8 +52,30 @@ The process of development of new content is open and transparent, using git and
 >
 {: .agenda}
 
+# Create a Github account, if you have not already
+
+> <hands-on-title>Create a Github account</hands-on-title>
+>
+> 1. Go to the [Github](https://github.com)
```

Comment:
```suggestion
> 1. Go to the [GitHub](https://github.com)
```
Is it really "the" GitHub, is that how people say it? They very possibly do but I don't recall hearing it said that way, more like "Go to GitHub" or "Go to the GitHub website".

Reply:
Yes!